lib/Mojo/Collection.pm

```diff
@@ -9,12 +9,12 @@ use Scalar::Util 'blessed';
 
 our @EXPORT_OK = ('c');
 
-sub TO_JSON { [@{shift()}] }
+sub TO_JSON { [@{CORE::shift()}] }
```

Comment:
I’m not sure what is best to read, but maybe many of these CORE::shift calls could be replaced by $_[0]?